src/main/groovy/de/dkfz/roddy/execution/jobs/BEJob.groovy

```diff
@@ -292,6 +292,6 @@ class BEJob<J extends BEJob> {
 
     @Override
     String toString() {
-        return "BEJob: ${jobName} calling tool ${tool.getAbsolutePath()}"
+        return "BEJob: ${jobName} calling tool ${tool?.getAbsolutePath() ?: getToolScript()}"
```

Comment:
This can be really bad. I'd prefer a two line version like:
if(getToolScript() {
  return "BEJob ... with piped script:\n\t" + getToolScript().readLines().join("\n\t")
} else {
  return "BEJob ... calling tool ${tool...}"
}

With this version, you'll have a clear multiline string for the piped script, whereas you'll have a one liner for the tool